mbed-src/config-headers/mbed_config.h

```diff
@@ -1,83 +1,2 @@
-#define MBED_CONF_PLATFORM_STDIO_CONVERT_TTY_NEWLINES                     0               // set by library:platform
-#define MBED_CONF_PLATFORM_STDIO_FLUSH_AT_EXIT                            1               // set by library:platform
-#define MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY                     0               // set by library:platform
-#define MBED_CONF_PLATFORM_USE_MPU                                        1               // set by library:platform
-#define MBED_CONF_RTOS_API_PRESENT                                        1               // set by library:rtos-api
-#define MBED_CONF_RTOS_IDLE_THREAD_STACK_SIZE                             512             // set by library:rtos
-#define MBED_CONF_RTOS_IDLE_THREAD_STACK_SIZE_DEBUG_EXTRA                 0               // set by library:rtos
-#define MBED_CONF_RTOS_IDLE_THREAD_STACK_SIZE_TICKLESS_EXTRA              256             // set by library:rtos
-#define MBED_CONF_RTOS_MAIN_THREAD_STACK_SIZE                             4096            // set by library:rtos
-#define MBED_CONF_RTOS_PRESENT                                            1               // set by library:rtos
-#define MBED_CONF_RTOS_THREAD_STACK_SIZE                                  4096            // set by library:rtos
-#define MBED_CONF_RTOS_TIMER_THREAD_STACK_SIZE                            768             // set by library:rtos
-#define MBED_CONF_TARGET_BOOT_STACK_SIZE                                  0x400           // set by library:rtos[*]
-#define MBED_CONF_TARGET_CONSOLE_UART                                     1               // set by target:Target
-#define MBED_CONF_TARGET_DEEP_SLEEP_LATENCY                               0               // set by target:Target
-#define MBED_CONF_TARGET_INIT_US_TICKER_AT_BOOT                           0               // set by target:Target
-#define MBED_CONF_TARGET_MPU_ROM_END                                      0x0fffffff      // set by target:Target
-#define MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE                   ETHERNET        // set by target:LPC1768
-#define MBED_CONF_TARGET_TICKLESS_FROM_US_TICKER                          0               // set by target:Target
-#define MBED_CONF_TARGET_US_TICKER_TIMER                                  3               // set by target:LPC1768
-#define MBED_CONF_TARGET_XIP_ENABLE                                       0               // set by target:Target
-#define MBED_STACK_DUMP_ENABLED                                           0               // set by library:platform
-// Macros
-#define _RTE_                                                                             // defined by library:rtos
-
-#endif
+// Just a proxy header. MBed's CMakeLists.txt will look for this file directly, so we need to tell it where to find the generated config
+#include "../../../mbed-cmake-config/config-headers/mbed_config_generated.h"
```

Comment:
This... really shouldn't be needed.  You should just be able to edit [this line](https://github.com/USCRPL/mbed-cmake/blob/a3ccad16edcca358d90fcde43555ad0f84c52135/mbed-src/CMakeLists.txt#L24) to point to the new locations of these headers.

Reply:
Okay I'll give it a go!